packages/mermaid/src/styles.ts

```diff
@@ -1,7 +1,7 @@
 import type { FlowChartStyleOptions } from './diagrams/flowchart/styles.js';
 import { log } from './logger.js';
 
-const themes: Record<string, any> = {};
+const themes: Record<string, (options?: any) => string> = {};
```

Comment:
`(options?: any) => string` Can be moved to a common `type`, which we can use here and in the styles of diagrams.

Reply:
Done.